### DISCUSSION TITLE
New team members

### DISCUSSION THREAD

**Discussion by kebekus** [Akaflieg-Freiburg/enroute#294] [Team Posts]
Hello,

a number of new people have joined the development team. I have therefore started to setup GitHub Teams and use this message to see how that works. Perhaps you could give me a short notice wether you received this message?

Here is the current state of affairs:

- Johannes Zellner and Heiner Tholen have volunteered to work on an autmated flight log.

- Markus Sachs has kindly translated to app to German

- Adrien Crovato has improved the GUI (The page that used to be called "Nearby Airfields" now contains more information). He will translate the app to French and work on METAR/TAF access.

- Michael Hrobaschk has kindly written a user manual, which will be accessible from the next version of the app.

- I will try to keep an overview, and work on bugfixing, GUI streamlining, weekly map generation, release management and on the upcoming desktop version.

As you can see, things are moving in the right direction. Thanks to all of you!

Best wishes from Freiburg,

Stefan.

**Comment by woodwind-enr** [comment]
Hallo Stefan,

ist angekommen.

Falls das user manual auf englisch geschrieben ist, kann ich gern eine 
deutsche Fassung anfertigen.

VG

M.


Am 25.06.2020 um 07:53 schrieb Stefan Kebekus:
